scripts/quick-helm-install.sh

```diff
@@ -0,0 +1,111 @@
+#!/usr/bin/env bash
+set -euo pipefail
+
+SMALL_INSTALL=false
+
+case "${1:-}" in
+    -h|--help)
+        echo -e "Usage:\n\tquick-helm-install.sh [options]"
+        echo " "
+        echo "Installs StackRox via Helm charts, requires Helm."
+        echo " "
+        echo "options:"
+        echo "-h, --help                show brief help"
+        echo "-s, --small               reduce StackRox resource requirements for small clusters"
+        exit 0
+        ;;
+    -s|--small)
+        SMALL_INSTALL=true
+        ;;
+esac
+
+echo "Adding the stackrox/helm-charts/opensource repository to Helm."
+
+helm repo add stackrox https://raw.githubusercontent.com/stackrox/helm-charts/main/opensource/
+
+echo "Generating STACKROX_ADMIN_PASSWORD"
+
+STACKROX_ADMIN_PASSWORD="$(openssl rand -base64 20 | tr -d '/=+')"
+
+echo "Installing stackrox-central-services"
+
+helm install -n stackrox --create-namespace stackrox-central-services stackrox/stackrox-central-services --set central.adminPassword.value="${STACKROX_ADMIN_PASSWORD}"
+
+if [[ "$SMALL_INSTALL" == "true" ]]; then
+    echo "Upgrading stackrox to a small install"
+    helm upgrade -n stackrox stackrox-central-services stackrox/stackrox-central-services \
+      --set central.resources.requests.memory=1Gi \
+      --set central.resources.requests.cpu=1 \
```

Comment:
I am having an issue when using the scrip locally. My central refuses to start with error `Warning  FailedScheduling  4m24s                 default-scheduler  0/1 nodes are available: 1 Insufficient cpu.`. All other services are running well. Maybe central requires more CPU?

Reply:
Sorry, my comment refers to the non-small version of the script 🤦

Reply:
The run with `--small` was okay and I could login by clicking on the link -  super cool!

I will try again with non-small

Reply:
ok, this will never work for non small. I have 4 CPU assigned to colima. 3 are used by 3 scanner instances, 0.2 is used for scanner-db, and there is only 0.8 CPU left for central. However central requires full 1 CPU, so it never starts.

I suggest to maybe inform the user, that their docker/k8s instance must have at least 4.2 CPUs resources to install central in the non-small version.

Reply:
Added a sentence to the readme regarding CPU resources.